packages/contracts/test/integration/commitReveal.spec.ts

```diff
@@ -104,27 +82,25 @@ describe("CommitRevealApp", async () => {
   it("should complete a full lifecycle", async function() {
     this.timeout(4000);
 
-    const startBalanceA = await A.getBalance();
-    const startBalanceB = await B.getBalance();
+    const startBalanceA = await alice.getBalance();
+    const startBalanceB = await bob.getBalance();
+
     // 1. Deploy & fund multisig
-    const multisig = new Multisig([A.address, B.address]);
+    const multisig = new Multisig([alice.address, bob.address]);
     await multisig.deploy(masterAccount);
     await masterAccount.sendTransaction({
       to: multisig.address,
-      value: Utils.UNIT_ETH.mul(2)
+      value: ethers.utils.parseEther("2")
```

Comment:
Passing thought: maybe we can have a wrapper around this to abstractly indicate 2 eth being used instead of having a utility function needing to parse it?

Reply:
What do you mean? `EtherAmount(2)`?

Reply:
Yeah something like that - def not a blocker by any means tho

Reply:
I'm not sure that's an abstraction worth doing. Esp if all the other Ethereum tooling uses raw integers.

Reply:
Yeah it may add unneeded overhead to existing usable patterns. Again, just a passing thought - not worth reading too much into

Reply:
Maybe just a shorthand util `const str2eth = x => ethers.utils.parseEther`